examples/sundials/ex10.cpp

```diff
@@ -4,16 +4,14 @@
 // Compile with: make ex10
 //
 // Sample runs:
-//    ex10 -m ../../data/beam-quad.mesh -r 2 -o 2 -s  5 -dt 0.15 -vs 10
-//    ex10 -m ../../data/beam-tri.mesh  -r 2 -o 2 -s  7 -dt 0.3  -vs 5
-//    ex10 -m ../../data/beam-hex.mesh  -r 1 -o 2 -s  5 -dt 0.2  -vs 5
+//    ex10 -m ../../data/beam-quad.mesh -r 2 -o 2 -s 12 -dt 0.1  -vs 10
```

Comment:
I see that `dt` had to be decreased here -- is this due to changed behavior/defaults in CVode? I'd expect values that used to work with previous versions to continue to work with the new version of SUNDIALS.

Reply:
@v-dobrev This change is due to some logic in the new linear solver interface that was bypassed by the old linear solver interface when supplying an external linear solver. If the initial residual of the linear system in the Newton solve is sufficiently small then the linear solve is skipped to save the cost of the solve. In this test, skipping the linear solve results in a smaller maximum step size. I suspect this would not be an issue with tighter tolerances but we could also add a user-callable set routine to disable this check.

Reply:
Veselin is sounds like a minor behavior change that is reasonable.  Do you have a problem with this?  We need to get this pulled in ASAP for the upcoming XSDK release.

Reply:
@acfisher, after some offline discussions with @gardner48 we found an appropriate solution and I already pushed the related updates. In other words, this issue has been resolved.

Reply:
Excellent.  Let's go ahead and get it pulled into master then.  Can you hit the approve button?